akka-stream/src/main/boilerplate/akka/stream/FanOutShape.scala.template

```diff
@@ -27,8 +27,8 @@ abstract class FanOutShape[-I] private (_in: Inlet[I @uncheckedVariance], _regis
   def this(init: FanOutShape.Init[I]) = this(init.inlet, init.outlets.iterator, init.name)
   
   final def in: Inlet[I @uncheckedVariance] = _in
-  final override def outlets: immutable.Seq[Outlet[_]] = _outlets
-  final override def inlets: immutable.Seq[Inlet[_]] = in :: Nil
+  override def outlets: immutable.Seq[Outlet[_]] = _outlets
```

Comment:
Please also add a scaladoc here saying "Not meant for overriding outside of Akka"